Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -1426,4 +1454,22 @@ getSelectedEstimatesObject :: String -> Maybe ChooseVehicle.Config
 getSelectedEstimatesObject dummy =
   case runExcept (decodeJSON (getValueToLocalStore ESTIMATE_DATA) :: _ ChooseVehicle.Config) of
     Right res -> Just res
-    Left err -> Nothing
\ No newline at end of file
+    Left err -> Nothing
+
+getChatSuggestions :: ST.HomeScreenState -> Array String
+getChatSuggestions state = do 
+  let didDriverMessage = (HU.didDriverMessage FunctionCall)
+      canShowSuggestions = case (DA.last state.data.messages) of 
+                          Just value -> ((not $ value.sentBy == "Customer") || not didDriverMessage)
```

Comment:
No need of so many brackets
`not (value.sentBy == "Customer") || not didDriverMessage`

Reply:
@PraveenGongada can't we use `value.sentBy /= "Customer"` instead of using `not $ value.sentBy == "Customer"`